Dockerfile

```diff
@@ -1,17 +1,30 @@
-FROM mcneilco/tomcat-maven:openjdk8
+ARG 	CHEMISTRY_PACKAGE=jchem
+ARG 	TOMCAT_IMAGE=mcneilco/tomcat-maven:1.3-openjdk8
+
+FROM 	${TOMCAT_IMAGE} as dependencies
+ARG     CHEMISTRY_PACKAGE
+ENV     CHEMISTRY_PACKAGE=${CHEMISTRY_PACKAGE}
+
+FROM 	dependencies as jchem
+ADD 	lib/jchem-16.4.25.0.jar /lib/jchem-16.4.25.0.jar
+RUN     mvn install:install-file -Dfile=/lib/jchem-16.4.25.0.jar -DartifactId=jchem -DgroupId=com.chemaxon -Dversion=16.4.25.0 -Dpackaging=jar -DgeneratePom=true -DcreateChecksum=true
+
+FROM 	dependencies as indigo
+
+FROM 	${CHEMISTRY_PACKAGE} as compile
 WORKDIR /src
-ENV CATALINA_HOME /usr/local/tomcat
-ENV PATH $CATALINA_HOME/bin:$PATH
 ADD 	pom.xml /src/pom.xml
-ADD    lib/jchem-16.4.25.0.jar /lib/jchem-16.4.25.0.jar
-RUN    ["mvn", "install:install-file","-Dfile=/lib/jchem-16.4.25.0.jar","-DartifactId=jchem","-DgroupId=com.chemaxon","-Dversion=16.4.25.0","-Dpackaging=jar","-DgeneratePom=true","-DcreateChecksum=true"]
-RUN    ["mvn", "dependency:resolve", "-P", "default"]
-RUN		["mvn", "clean"]
-ADD		. /src
-RUN		mvn compile war:war -P default
-RUN		mv target/acas*.war $CATALINA_HOME/webapps/acas.war
-RUN		mv target/acas* $CATALINA_HOME/webapps/acas
-RUN		rm -rf /src
-WORKDIR	$CATALINA_HOME
-EXPOSE	8080
-CMD		["catalina.sh", "run"]
\ No newline at end of file
+RUN     mvn dependency:resolve-plugins
+RUN 	mvn dependency:resolve -P ${CHEMISTRY_PACKAGE}
+ADD 	. /src
+RUN 	mvn clean && \
+        mvn compile war:war -P ${CHEMISTRY_PACKAGE} && \
+        mv target/acas*.war $CATALINA_HOME/webapps/acas.war && \
+        mv target/acas* $CATALINA_HOME/webapps/acas
+
+FROM 	${TOMCAT_IMAGE} as build
+COPY 	--from=compile /src/target/acas*.war $CATALINA_HOME/webapps/acas/ $CATALINA_HOME/webapps/acas.war
+COPY 	--from=compile /src/target/acas* $CATALINA_HOME/webapps/acas/ $CATALINA_HOME/webapps/acas
+WORKDIR $CATALINA_HOME
+EXPOSE 	8080
+CMD 	["catalina.sh", "run"]
```

Comment:
- Use newer mcneilco/tomcat-maven:1.3-openjdk8 build (minor tomcat bump to patch vulnerabilities and upgrade to centos8)
  - https://bitbucket.org/mcneilco/tomcat-maven/commits/1d9fcbfba56e6cfb8d1d64dd8e8ed4b0a3a20d23
- Multi stage build to remove .m2 directory from final build.
- Consolidates the indigo and jchem Docker files

Reply:
I see, we are only copying the .war and the expanded directory into Tomcat and nothing else.